src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/aligndp/wfsyncstandin/SynchronizeSyncAcaWFImpl.java

```diff
@@ -0,0 +1,110 @@
+    SynchronizeDebtPositionWfConfig wfConfig = applicationContext.getBean(SynchronizeDebtPositionWfConfig.class);
+    synchronizeInstallmentAcaActivity = wfConfig.buildSynchronizeInstallmentAcaActivity();
+    finalizeDebtPositionSyncStatusActivity = wfConfig.buildFinalizeDebtPositionSyncStatusActivityStub();
+    sendDebtPositionIONotificationActivity = wfConfig.buildSendDebtPositionIONotificationActivityStub();
+  }
+
+  @Override
+  public void synchronizeDPSyncAca(DebtPositionDTO debtPosition) {
+    Map<String, IupdSyncStatusUpdateDTO> iupdSyncStatusUpdateDTOMap = invokeAcaStandInCreateDebtPositionActivity(debtPosition);
+
+    DebtPositionDTO debtPositionDTO = finalizeDebtPositionSyncStatusActivity.finalizeDebtPositionSyncStatus(debtPosition.getDebtPositionId(), iupdSyncStatusUpdateDTOMap);
+    log.info("Sync status updated for IUD/IupdPagoPa and new statuses: {}", iupdSyncStatusUpdateDTOMap);
+
+    sendDebtPositionIONotificationActivity.sendMessage(debtPositionDTO);
+    log.info("Message sent to IO for organizationId {} and debtPositionTypeOrgId {}", debtPositionDTO.getOrganizationId(), debtPositionDTO.getDebtPositionTypeOrgId());
+  }
+
+  private Map<String, IupdSyncStatusUpdateDTO> invokeAcaStandInCreateDebtPositionActivity(DebtPositionDTO debtPosition) {
+    Map<String, IupdSyncStatusUpdateDTO> syncStatusMap = new HashMap<>();
+
+    log.info("Starting workflow for synchronizing DebtPosition on ACA with ID: {}", debtPosition.getDebtPositionId());
+
+    debtPosition.getPaymentOptions().forEach(paymentOption ->
+      paymentOption.getInstallments().stream()
+        .filter(installment -> InstallmentDTO.StatusEnum.TO_SYNC.equals(installment.getStatus()))
+        .forEach(installment ->
+          synchronizeInstallmentAca(debtPosition, installment.getIud(), syncStatusMap, Objects.requireNonNull(installment.getSyncStatus()).getSyncStatusTo()))
+    );
+
+    return syncStatusMap;
+  }
+
+  private void synchronizeInstallmentAca(DebtPositionDTO debtPosition, String iud, Map<String, IupdSyncStatusUpdateDTO> syncStatusMap, InstallmentSyncStatus.SyncStatusToEnum newStatus) {
+    try {
+      log.info("Synchronizing Installment with IUD: {} for DebtPosition ID: {}", iud, debtPosition.getDebtPositionId());
+
+      synchronizeInstallmentAcaActivity.synchronizeInstallmentAca(debtPosition, iud);
+
+      updateSyncStatusMap(IupdSyncStatusUpdateDTO.NewStatusEnum.valueOf(newStatus.name()), syncStatusMap, iud);
+    } catch (Exception e) {
+      log.error("Error occurred while synchronizing Installment with IUD: {} for DebtPosition ID: {}. Error: {}",
+        iud, debtPosition.getDebtPositionId(), e.getMessage());
+
+      paymentsProducerService.notifyPaymentsEvent(
+        debtPosition,
+        PaymentEventType.SYNC_ERROR,
+        e.getMessage()
+      );
+
+      updateSyncStatusMap(IupdSyncStatusUpdateDTO.NewStatusEnum.TO_SYNC, syncStatusMap, iud);
```

Comment:
Don't update status of errore, this way It Will stay TO_SYNC and syncStatus Will be not deleted